service/src/test/java/tds/session/web/endpoints/ExternControllerIntegrationTests.java

```diff
@@ -0,0 +1,44 @@
+package tds.session.web.endpoints;
+
+import com.jayway.restassured.http.ContentType;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.IntegrationTest;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import tds.session.SessionServiceApplication;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = SessionServiceApplication.class)
+@WebAppConfiguration
+@IntegrationTest("server.port:8080")
+public class ExternControllerIntegrationTests {
+    @Test
+    public void shouldReturnExtern() {
+        given()
+            .accept(ContentType.JSON)
+            .when()
```

Comment:
I think IntelliJ "fixed" the tabs at some point so when() and then() are not one more tab to the left.  I wonder if there is a way to tell IntelliJ to not do that in certain cases, cause it seems like we will keep battling IntelliJ on this one.